src/open_inwoner/scss/components/Contactmomenten/Contactmomenten.scss

```diff
@@ -74,46 +74,21 @@
 }
 
 .contactmoment {
-  &__details {
-    // Set table width for word-break
-    table {
-      margin-top: 0;
-      overflow-x: auto;
-      table-layout: fixed;
-      width: 100%;
-      @media (min-width: 768px) {
-        width: var(--mobile-ms-width);
-      }
-    }
-    table th {
-      width: 100px;
-      white-space: nowrap;
-    }
-    table td {
-      width: 300px;
-      overflow: hidden;
-      word-break: break-word;
-      white-space: normal;
-
-      &.table__item--status {
-        .table__item--wrapper {
-          display: flex;
-          justify-content: space-between;
-        }
-      }
+  p {
```

Comment:
Do not style tags without classes - we only do that if we do not have control over the components (like with CKeditor) - so your contactmoment__text class is perfectly fine , and just add the color in there.